RecoTracker/TkSeedGenerator/python/SeedGeneratorFromProtoTracksEDProducer_cff.py

```diff
@@ -12,7 +12,7 @@
 pixelTTRHBuilderWithoutAngle = copy.deepcopy(ttrhbwr)
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelTriplets_cfi import *
 from RecoTracker.TkSeedingLayers.PixelLayerTriplets_cfi import *
-from RecoPixelVertexing.PixelTrackFitting.PixelTracks_cfi import *
+from RecoPixelVertexing.PixelTrackFitting.pixelTracksDefault_cfi import *
```

Comment:
@makortel 
Please you do that part after this PR as I am not the expert in this... Thanks!